michael/express/lab01/app.js

```diff
@@ -0,0 +1,45 @@
+const express = require("express"); // Import express
+const app = express(); // Create an instance of express
+
+// Defines port, optionally from environment variable
+const port = process.env.PORT || 3333;
+
+// Array of messages
+const messages = [
+	"Try Again",
+	"You are wrong",
+	"You are right",
+	"No",
+	"Yes",
+	"Maybe",
+	"I don't know",
+];
+
+// Choose a random message from messages
+const randomMessage = () => {
+	return messages[Math.floor(Math.random() * messages.length)];
+};
+
+// Define Routes
+// Root route
+app.get("/", (req, res) => {
+	console.log("GET /");
```

Comment:
This is also a nice trick to see which route is being used

Reply:
We will add in an express addon to take care of this in the future too!

Reply:
Oh, nice @ the addon. I should've looked for that in the first place probably.

I actually don't even know the best way to find node packages. But I suppose maybe this package? https://www.npmjs.com/package/express-route-logging